src/pdn/src/connect.cpp

```diff
@@ -925,22 +925,22 @@ void Connect::recordFailedVias() const
     std::string reason_str;
     switch (reason) {
       case failedViaReason::OBSTRUCTED:
-        reason_str = "Obstructed";
+        reason_str = "Via - Obstructed";
```

Comment:
@maliberty the vias could also be in their own category under the tool, what do you think?

Reply:
I think that makes sense